packages/gatsby-plugin-vercel-builder/src/handlers/build.ts

```diff
@@ -88,10 +87,6 @@ export async function copyFunctionLibs({
         src: join('.cache', 'page-ssr'),
         dest: join(functionDir, '.cache', 'page-ssr'),
       },
-      // {
-      //   src: join(functionDir, '.cache', 'query-engine', 'assets'),
-      //   dest: join(functionDir, 'assets'),
-      // },
```

Comment:
Should this comment be removed?

Reply:
Ya, was just cleaning up some dead code while in here.

Reply:
According to https://github.com/vercel/vercel/commit/ae89b8b8bee068cf9499867d6c794cb2ccc47c24#diff-3b89128206f28715de44e9a753ba57ac89f9ea0b484035ca0d74c0a3f8d9204bR98-R101, these lines have always been commented out. Sounds like they're not needed.